backend/inspirehep/records/marshmallow/literature/latex.py

```diff
@@ -42,23 +42,27 @@ def get_author_names(self, data):
 
     def get_publication_info(self, data):
         publication_info = BibTexCommonSchema.get_best_publication_info(data)
```

Comment:
it would have been simpler to add a `.copy()` here rather than renaming everything.